deps/build.jl

```diff
@@ -151,7 +151,8 @@ const gcc_support = (
     (v"6.5", v"4.8.2"),
     (v"7.0", v"4.9.2"),
     (v"7.5", v"4.9.2"),
-    (v"8.0", v"5.3.1") )
+    (v"8.0", v"5.9")) # Before 8.0.61 only up to 5.3.1 was supported, since then 5.X is supported
```

Comment:
I've been basing that check on the CUDA installation guide, so has probably been `>5` all along (with that comment likely being wrong).

Maybe document that those last two are (correctly) based on `host_config.h`, while the others are from the installation guide (which presumable only lists verified set-ups).

Also, you can compare against `v"6.0-"` to match all of GCC 5.